### ISSUE TITLE
Red Code Text

### ISSUE THREAD

**Issue by crguendelsberger** [Sandrem/FlyCasual#1668]
It looks like this issue has been brought up before but just for the sake of continuity, I'll report it again; perhaps it may help you continue to incorporate refinements.

I'm brand new to GitHub. Disclaimer: I only signed up for GitHub because I had to in order to submit this bug report to you, but since I think what you're doing is so cool and so amazing and think it has a lot of potential, I not only created a GitHub account (which I'll probably never use except in correspondence to you), I am also happily supporting your efforts on Patreon.  I have zero programming background and had to be walked through how to install Adobe by our organization IT functional yesterday.  I am a simple aviator.  Push yoke forward, houses get bigger.  Pull yoke back, houses get smaller.  

I'm glad there are people like you out there with skills I could only dream of having.  I only hope I attached the screenshot correctly for your analysis.

![Slide1](https://user-images.githubusercontent.com/48625097/54477033-c7f57500-47d1-11e9-95be-cf815228cd73.JPG)

**Comment by rune-b** [comment]
Thanks for reporting. It would be helpful if you write a short description of what was going on in the game when the error appeared.

From the screenshot it looks like it happened when Miranda Doni attacked, is this correct? And if so, was it before or after you chose to use her ability?

**Comment by crguendelsberger** [comment]
It must have been after she used her ability - she was part of the AI squad
that I had built.

**Comment by rune-b** [comment]
It looks like it has something to do with the shot being obstructed by an asteroid, but I can't reproduce this bug. Has this happened for you more than once?

**Comment by crguendelsberger** [comment]
Yes, it happened once before but I don't recall the specifics at the time.
I've also had AI opponents inexplicably flee from the mat, which was really
weird. They weren't ionized or tractored or anything.

(701) 340-9484 is my cell.

Awesome job dude! This will be my favorite app. I would pay to play it's so
good but at least I can be a patreon sponsor. I can't wait for it to come
out on iOS!

**Comment by crguendelsberger** [comment]
Hey, how do I update the game? Is that automatic or do I need to download
the client again from GitHub?

**Comment by Sandrem** [comment]
@crguendelsberger You will see notification in main menu - "new version is available". Click on this notification, github page will be opened, you will need to download new version from that page.

**Comment by crguendelsberger** [comment]
You mentioned in the Patreon email that you have a lot of plans for the
simulator but are limited on resources (aren't we all lol).
You have 58 patrons at this point.  It sounds like you could implement your
plans if everyone increased their contribution by "x".  What does "x"
equal?

**Comment by Sandrem** [comment]
@crguendelsberger At the moment I have a regular job, and develop the game like hobby, in my free time after work or on holidays, ~10 hours per week. This month I was too busy at work, so I coudn't spend even 50% of my usual time for new update. In the dream-world, I would like to work as full-time developer (~80 hours per week) and to live by funds from Patreon, but current funds are like ~1/6 of salary of programmer in my region, and last 6 months they are almost at he same level - seems that I reached maximum, community is not so big. But I think that with x3 or even x2 funds I could find a more simple job to get ~40 hours per week of free time to spend on my project.
Also, I am a bit disappointed about support from another developers. I got a great help in bugfixing and development of new cards, but I don't have almost any support in Unity-specific areas, like UI or online multiplayer, and I have a too big list of planned features, where I have very low chances to have any help from another developers. As example, I abandoned online multiplayer until I will have more support from Patreon or another developers.
In any way, current support is exceeded my initial expectations and I will continue development and plan some sweet features in next few months, like survival mode or missions =)

**Comment by crguendelsberger** [comment]
Hey I came across this anomaly tonight playing on Android. I had just
activated Kraken.

**Comment by crguendelsberger** [comment]
Here's some recommendations, some of which you may have already implemented:

More/different types of obstacles (gas clouds, debris clouds).

Inclusion of the strain token.

Scrub available pilot/upgrade cards and ensure each ship has the approved
number/type of each.

Slow down the dice throws. It's crazy fast and hard to follow when one of
your ships is being attacked by AI.

More to follow but those are the big ones that come to mind.

Keep up the excellent work my friend.

Chad

**Comment by bitsai** [comment]
AFAICT, the specific issues/suggestions here have either been implemented or are already tracked elsewhere, closing for now.
